Kentor.AuthServices.Tests/WebSSO/LogoutCommandTests.cs

```diff
@@ -204,6 +213,28 @@ public void LogoutCommand_Run_ChecksForLocalReturnUrl()
         }
 
         [TestMethod]
+        public void LogoutCommand_Run_Calls_NotificationForAbsoluteUrl()
```

Comment:
Would it make sense to add a test (or modify an existing test) to verify that the notification is **not** called for a relative url? This would formalize in the test suite what's currently just in our PR conversations.